README.md

```diff
@@ -69,3 +69,14 @@ pre-commit install
 # To disable:
 # pre-commit uninstall
 ```
+
+Tests should now pass:
+
+```sh
+make check-tests
+make check-types
+make check-lint
+make check-format
```

Comment:
These are all helper targets under the umbrella `make check`, which is what I'd recommend documenting here.

In general, we have a little helper target that prints top-level targets that is printed if you call `make` without arguments.

Reply:
For local dev work, I use `make check` to replicate a "fast" version of CI checks and `make check-fix` to do the same but to apply formatting changes rather than just erroring.

Reply:
I'll mention those instead. Thanks!